### PR TITLE
Gtfs trips to NTM vehicle journeys and trip properties

### PR DESCRIPTION
**Pull request by datanel**
from GTFS trips generate 
- Vehicle journeys
- Trip properties

related to https://github.com/CanalTP/navitia_model/pull/57

### CODE REVIEW

File: src/gtfs/read.rs [hunk 1]
```diff
@@ -465,9 +513,9 @@ fn get_route_with_smallest_name<'a>(routes: &'a [&Route]) -> &'a Route {
 }
 
 type MapLineRoutes<'a> = HashMap<(Option<String>, String), Vec<&'a Route>>;
-fn map_line_routes(gtfs_routes: &[Route]) -> MapLineRoutes {
+fn map_line_routes(gtfs_routes: &CollectionWithId<Route>) -> MapLineRoutes {
     let mut map = HashMap::new();
-    for r in gtfs_routes {
+    for r in gtfs_routes.iter().map(|(_, r)| r) {
```
[the remark this answers is not in the document]
Comment (datanel) [reply src/gtfs/read.rs]: @TeXitoi why `std::slice::Iter`?

File: src/gtfs/read.rs [hunk 2]
```diff
@@ -1039,29 +1028,23 @@ mod tests {
...
             assert_eq!(2, collections.lines.len());
 
-            let mut lines_ids: Vec<String> = collections
-                .lines
-                .iter()
-                .map(|(_, l)| l.id.clone())
-                .collect();
+            let mut lines_ids: Vec<String> =
+                collections.lines.values().map(|l| l.id.clone()).collect();
             lines_ids.sort();
```
Comment (TeXitoi) [inline src/gtfs/read.rs]: This deserve a `fn extract_ids<T: Id<T>>(c: &Collection<T>) -> Vec<String>`

Comment (datanel) [reply src/gtfs/read.rs]: absolutely
in another PR?

Comment (TeXitoi) [reply src/gtfs/read.rs]: OK

**Pull request merged**